reactivesocket-core/src/main/java/io/reactivesocket/Frame.java

```diff
@@ -562,9 +564,37 @@ public String toString() {
             }
 
             streamId = FrameHeaderFlyweight.streamId(directBuffer, 0);
+
+            switch (type) {
+            case LEASE:
+                additionalFlags = " Permits: " + Lease.numberOfRequests(this) + ", TTL: " + Lease.ttl(this);
```

Comment:
This is a bit inconsistent as you don't use , as a separator in the normal toString values